herd/parseTest.ml

```diff
@@ -275,78 +277,90 @@ module Top (TopConf:Config) = struct
 (* Markers START/END below are for excluding source when compiling
    the web interface *)
 (* START NOTWWW *)
+         let is_morello = Conf.variant Variant.Morello in
+         let module ConfMorello =
+           struct let is_morello = is_morello end in
          if Conf.variant Variant.Morello then
-           let module X = AArch64Make(CapabilityValue) in
+           let module
+               AArch64Value = CapabilityValue.Make(ConfMorello) in
+           let module X = AArch64Make(AArch64Value) in
            X.run dirty start_time name chan env splitted
          else if Conf.variant Variant.Neon then
-           let module X = AArch64Make(Uint128Value) in
+           let module AArch64Value = Uint128Value.Make(ConfMorello) in
+           let module X = AArch64Make(AArch64Value) in
            X.run dirty start_time name chan env splitted
          else
 (* END NOTWWW *)
+           let module AArch64Value = AArch64Value.Make(ConfMorello) in
            let module X = AArch64Make(AArch64Value) in
            X.run dirty start_time name chan env splitted
```

Comment:
nit: this is common across the if ... then else if ... else ... above.

Reply:
Indeed but I cannot see how to avoid code repetition, without using modules as values. Another solution would be for the AArch64Make to performe the call to its own X.run with all arguments. Something like:
```OCaml
let module MakeAArch64(...) = struct
  ...
   let run () = C.run dirty start_time name chan env splitted
end
```
Doing so, we would avoid repeating the list of arguments twice.

None of these coding is very appealing to me...